src/rebar_prv_compile.erl

```diff
@@ -103,3 +107,23 @@ handle_args(State) ->
     {Args, _} = rebar_state:command_parsed_args(State),
     Jobs = proplists:get_value(jobs, Args, ?DEFAULT_JOBS),
     {ok, rebar_state:set(State, jobs, Jobs)}.
+
+copy_app_dirs(AppName, OldAppDir, AppDir) ->
+    case ec_cnv:to_binary(filename:absname(OldAppDir)) =/=
+        ec_cnv:to_binary(filename:absname(AppDir)) of
+        true ->
+            Name = ec_cnv:to_list(AppName),
+            AppFile = filename:join([OldAppDir, "ebin", Name++".app"]),
+            case filelib:is_file(AppFile) of
+                true ->
+                    file:copy(AppFile,
+                              filename:join([AppDir, "ebin", Name++".app"]));
+                false ->
+                    ok
+            end,
+            filelib:ensure_dir(filename:join(AppDir, "dummy")),
+            [file:make_symlink(filename:join(OldAppDir, Dir), filename:join(AppDir, Dir))
+            || Dir <- ["priv", "include"]];
+        false ->
+            ok
+    end.
```

Comment:
I'll add links for src and if there are other src dirs set it can make links for those.

I really don't like the thought of copy :)